exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/ConvertCountToDirectScan.java

```diff
@@ -121,9 +121,9 @@ public void onMatch(RelOptRuleCall call) {
 
     final RelDataType scanRowType = constructDataType(agg, result.keySet());
 
-    final DynamicPojoRecordReader<Long> reader = new DynamicPojoRecordReader<>(
+    final DynamicPojoRecordReader<Double> reader = new DynamicPojoRecordReader<>(
```

Comment:
This will change return type for "select count(*) ...", let's keep it to `Long`.

Reply:
Yes, agree. Done.